lib/core/diagnostics.js

```diff
@@ -114,6 +120,74 @@ if (undiciDebugLog.enabled || fetchDebuglog.enabled) {
   isClientSet = true
 }
 
+// Track fetch requests
+if (fetchDebuglog.enabled) {
+  const debuglog = fetchDebuglog.enabled
```

Comment:
first bug I found is this:

```suggestion
  const debuglog = fetchDebuglog
```

Reply:
Yes, this was removed